roles/custom/matrix-media-repo/templates/media-repo/media-repo.yaml.j2

```diff
@@ -0,0 +1,359 @@
+# General repo configuration
+repo:
+  bindAddress: {{ matrix_media_repo_bind_address }}
```

Comment:
Adding ` | to_json` everywhere in this file is a good way to ensure that more special values do not cause breakage. It's probably unnecessary for most things, but it's a good best-practice which we try to follow in most roles.